client/web/src/integration/context.ts

```diff
@@ -10,6 +10,11 @@ import { SharedGraphQlOperations } from '../../../shared/src/graphql-operations'
 import html from 'tagged-template-noop'
 import { commonWebGraphQlResults } from './graphQlResults'
 
+export interface ServerSideEvent {
```

Comment:
I think SSE = Server _Sent_ Events https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events
```suggestion
export interface ServerSentEvent {
```

Reply:
Removed this and I'm now using the real type for search events.